api/v1/services/ProjectService.js

```diff
@@ -0,0 +1,208 @@
+var Checkit = require('checkit');
+var _Promise = require('bluebird');
+var _ = require('lodash');
+
+var Mentor = require('../models/Mentor');
+var Project = require('../models/Project');
+var ProjectMentor = require('../models/ProjectMentor');
+
+var errors = require('../errors');
+var utils = require('../utils');
+var roles = require('../utils/roles');
+
+const projectKeys = ["name", "description", "repo", "isPublished"];
```

Comment:
Why do you need this? The service's job is not to check for the existence of parameters -- that's the controller/request middleware's job.